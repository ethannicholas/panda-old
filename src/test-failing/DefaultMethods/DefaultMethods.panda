interface Interface1 {
    method hello() {
        Console.writeLine("Hello from interface1!")
    }
}

interface Interface2 {
    method hello() {
        Console.writeLine("Hello!")
    }

    method greet()
}

class Test (Interface1, Interface2) {
    @override
    method greet() {
        Console.writeLine("Also hello!")
    }

    @class
    method main() {
        var test := new Test()
        test.hello()
        test.greet()
    }
}