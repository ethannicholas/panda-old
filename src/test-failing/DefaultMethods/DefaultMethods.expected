Hello from interface1!
Also hello!
